### PR TITLE
Move contribution guide to a Markdown file

### PR DESCRIPTION
**Pull request by armands-malejevs**
#### Describe the changes
Move contribution guide to a `CONTRIBUTING.md` file and link to it from the readme. This will allow new contributors to find the guide more easily and everyone will be able to propose edits in the form of a PR.

Once merged this will deprecate the currently used document https://github.com/react-native-video/react-native-video/issues/2653

This addresses the following issue https://github.com/react-native-video/react-native-video/issues/2713

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Issues
+
+* New issues are reviewed and if they require additional work will be marked with the [`triage needed`](https://github.com/react-native-video/react-native-video/labels/triage%20needed) label. This is an open call for help from the community to verify the issue and help categorize it. If an issue stays in this state for a long time, it will be closed unresolved.
+* Once an issue has been reviewed it will be labeled with [`help wanted`](https://github.com/react-native-video/react-native-video/labels/help%20wanted) to indicate it is ready to be worked on. Please wait for this label before submitting a PR to avoid spending time on something that is likely to be rejected.
+
+## Cleanup
+
+* Given the history of this project, we are going to be more aggressive than usual in keeping things clean. We are working with limited resources and do not want to return to the 1000+ open issues state. This is meant to be disrespectful or hostile. It is just a way to keep the limited resources we have focused. If your issue was closed prematurely, just chime in and engage!
```
Comment (nbennink) [inline CONTRIBUTING.md]:
```suggestion
* Given the history of this project, we are going to be more aggressive than usual in keeping things clean. We are working with limited resources and do not want to return to the 1000+ open issues state. This is not meant to be disrespectful or hostile. It is just a way to keep the limited resources we have focused. If your issue was closed prematurely, just chime in and engage!
```
I hope so at least

Comment (armands-malejevs) [reply CONTRIBUTING.md]: Lol 😆 Thanks!

**Comment by hueniverse** [comment]
I fixed the issue today with this:
```
* Pull requests require 1-3 approved reviews to be merged.
* The number of reviews depends on the complexity by adding up (max of 3):
    * `1` reviewer for each PR 
    * `1` if more than 3 files and/or 30 lines of code changed
    * `1` for each native platform code changes involved
```

**Comment by armands-malejevs** [comment]
@hueniverse I updated to the latest contribution guide 👍

**Pull request merged**